themes/ee/asset/javascript/src/cp/form_validation.js

```diff
@@ -326,9 +326,14 @@ EE.cp.formValidation = {
 			grid = true;
 		}
 
+		if (fieldset.find('.fluid').size()>0)
```

Comment:
While we don't have any coding convertions for javascript code, the existing style is to have spaces around `>` so please add those.